### PR TITLE
Pro 5610 field indicator

### PR DESCRIPTION
**Pull request by ValJed**
[PRO-5610](https://linear.app/apostrophecms/issue/PRO-5610/ui-field-indicators)

## Summary

Make translation fields metadata showing up when a field has been translated.
New vue component handling meta logic to show or not `AposTranslationIndicator.vue`.

⚠️ Must be tested with [this PR](https://github.com/apostrophecms/automatic-translation/pull/7). And a fake provider must be added.

## What are the specific steps to test this change?

With fake provider, translate a document. In the new locales you should see the metadata indicators for supported fields.

## What kind of change does this PR introduce?

- [ ] Bug fix
- [X] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [X] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

### CODE REVIEW

File: modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue [hunk 1]
```diff
@@ -56,16 +56,16 @@
...
-              :namespace="component.namespace"
-              :meta="component.data"
+              :namespace="namespace"
+              :meta="data"
               :meta-raw="meta"
-              :data-apos-test-component="component.name"
-              :data-apos-test-namespace="component.namespace"
+              :data-apos-test-component="name"
+              :data-apos-test-namespace="namespace"
               data-apos-test="field-meta"
```
Comment (ValJed) [inline modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue:69]: Just destructuring for readability

File: modules/@apostrophecms/schema/ui/apos/logic/AposInputWrapper.js [hunk 2]
```diff
@@ -135,11 +135,11 @@ export default {
...
       return apos.schema.fieldMetadataComponents
-        .map(component => {
+        .map(({ name, namespace }) => {
           return {
-            name: component.name,
-            namespace: component.namespace,
-            data: meta[component.namespace] || {}
+            name,
+            namespace: namespace,
+            data: meta[namespace] || {}
```
Comment (ValJed) [inline modules/@apostrophecms/schema/ui/apos/logic/AposInputWrapper.js:142]: Just destructuring for readability

File: modules/@apostrophecms/translation/ui/apos/components/AposTranslationIndicator.vue [hunk 3]
```diff
@@ -30,7 +30,7 @@
 <script>
 
 export default {
-  name: 'AposTranslationChip',
+  name: 'AposTranslationIndicator',
```
Comment (ValJed) [inline modules/@apostrophecms/translation/ui/apos/components/AposTranslationIndicator.vue:33]: Forgot to rename here.

File: modules/@apostrophecms/translation/ui/apos/components/AposTranslationMeta.vue [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      }
+    }
+  },
+  data() {
+    return {
+      isShowed: this.meta.data?.changed
+    };
+  }
+};
+</script>
```
Comment (ValJed) [inline modules/@apostrophecms/translation/ui/apos/components/AposTranslationMeta.vue:37]: New component handling meta logic for translation. Uses already existing `AposTranslationIndicator.vue`.
Some props are unused here and could be removed. Wasn't sure if we are going to need them soon so I kept them.

**Review by myovchev — CHANGES_REQUESTED** [review]
The Meta component shouldn't exist  here.

**Review by myovchev — APPROVED** [review]

**Pull request merged**